### ISSUE TITLE
nested macro "Expands to"

### ISSUE THREAD

**Issue by loynoir** [microsoft/vscode-cpptools#10335]
### Reproduce
Nested macro
```c
#define X(a) Y(a)
#define Y(a) "hello world"
```

### Current
- Hover on `Y(a)` shows
```
Expands to:

"hello world"
```

- Hover on `X(a)` shows
```
Expands to:

Y(a)
```

### Feature Request
- Hover on `Y(a)` shows
```
Expands to:

"hello world"
```

- Hover on `X(a)` shows
```
Expands to:

Y(a)
```

If hover on `X(a)` while the key `ctrl` (or key `shift`, or any else special key) is pressed, show final expansion

```
Final Expands to:

"hello world"
```

**Comment by sean-mcmanus** [comment]
This is a duplicate of https://github.com/microsoft/vscode-cpptools/issues/3579. You can upvote that issue, although it's already our 2nd most upvoted IntelliSense-related feature request.

**Comment by sean-mcmanus** [comment]
A fix is available with 1.14.1 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.14.1
